fastrand.go

```diff
@@ -111,6 +108,9 @@ func Intn(n int) int {
 }
 
 // BigIntn returns a uniform random value in [0,n). It panics if n <= 0.
+//
+// TODO: This particular function has not been extended to be faster, as it is
+// just a passthrough.
```

Comment:
I don't think we can speed up `rand.Int`. It will be much faster since we pass in our own Reader instead of the crypto.rand Reader.
Benchmark:
```
BenchmarkBigIntn-4         	 3000000	       569 ns/op
BenchmarkBigIntnCrypto-4   	  300000	      4424 ns/op
```